scripts/discover_devices.py

```diff
@@ -39,44 +64,63 @@
             'power_consumption': 0,
             'remaining_time': '00:00:00'}
 
-"""  # noqa: E501
+Print all protocol types devices for 30 seconds:
+    python discover_devices.py 30 -t all\n
 
-from argparse import ArgumentParser
-from asyncio import get_event_loop, sleep
-from dataclasses import asdict
-from pprint import PrettyPrinter
+Print only protocol type 1 devices:
+    python discover_devices.py -t 1\n
 
-from aioswitcher.bridge import SwitcherBridge
-from aioswitcher.device import SwitcherBase
-
-printer = PrettyPrinter(indent=4)
+Print only protocol type 2 devices:
+    python discover_devices.py -t 2\n
+"""  # noqa E501
+)
 
-parser = ArgumentParser(description="Discover and print info of Switcher devices")
+parser = ArgumentParser(
+    description="Discover and print info of Switcher devices",
+    epilog=_examples,
+    formatter_class=RawDescriptionHelpFormatter,
+)
 parser.add_argument(
     "delay",
     help="number of seconds to run, defaults to 60",
     type=int,
     nargs="?",
     default=60,
 )
+possible_types = ["1", "2", "all"]
```

Comment:
the type1/2 is frustrating...
a better categorizing title would be much better.

Reply:
Same...